test/helpers/wrappers.go

```diff
@@ -46,7 +46,7 @@ func Ping6(endpoint string) string {
 // variadic optinalValues argument. This is passed on to fmt.Sprintf() and uses
```

Comment:
Nit: typo in commit: stadistics --> statistics

Reply:
done